server/src/controllers/company.ts

```diff
@@ -29,6 +31,44 @@ export async function getAllCompanies(
   res.json(companies);
 }
 
+/**
+ * Returns a companies that fits a certain company name and optionally branch name
+ * @param req
+ * @param res
+ * @param next
+ */
+export async function searchCompanyByName(
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<void> {
+  const user = await User.findOne({ emailAddress: req.user?.email }).lean().select("isAdmin");
+  if (!user) return next(new NotFound("User not found"));
+
+  const searchOptions: { [k: string]: any } = {};
+
+  if (req.query.companyName) {
```

Comment:
Ich würde die query parameter lowercase machen oder kebab case also company-name

Reply:
ah guter punkt danke

Reply:
@jakepanter es wäre praktischer es nicht zu machen. überall sonst verwende ich camel case. so geht es viel einfacher in der übertragung ins datenmodell, da da alles camel case ist. ansonsten wird der code sehr viel länger, in den anderen controller methoden.

Reply:
lass es erstmal so lassen und ich mach nen issue dafür auf

Reply:
ja kein problem das hat ja keine prio gerade ;)